docs/api.md

```diff
@@ -334,3 +346,212 @@ curl -iX POST --form file=@/path/to/splits_file.lss https://splits.io/api/v3/run
+    by including this header in your requests:
+    ```http
+    Bearer: OAuth YOUR_ACCESS_TOKEN
+    ```
+    The access token expires after the duration specified in `expires_in` (measured in seconds). After it expires, you
+    can retrieve a new one with no user intervention using redirection upon accessing your app, or a hidden iframe to
+    invisibly take the user through the authorization flow. If the user is still authorized, no user interaction will be
+    required and you can strip your new access token from the URL fragment.
+
+    This style of expiring access tokens periodically improves security by limiting the usability of any stolen
+    credentials.
+
+#### Example 3: My application is a website
+If your application is a website with a backend component, you should use OAuth's **authorization code grant flow**.
+This means your website will link the user to the Splits I/O authorization page, and if the user accepts the
+authorization, Splits I/O will give your application a `code` which it will immediately exchange for an OAuth token
+using a secure API request.
+
+1. On your Splits I/O [settings page][1], set your `redirect_uri` to where you want users to land after going through
+   the authorization flow. For this example, we'll use
+    ```http
+    https://YOUR_WEBSITE/auth/splitsio
+    ```
+    *Hint: Set this to "debug" for now if you don't yet have a page to redirect yourself to.*
+
+    Set your requested scopes to `upload_run`.
+2. When a user wants to grant authorization to your application for their Splits I/O account, send them to a URL like
+    this:
+    ```http
+    https://splits.io/oauth/authorize?response_type=code&scope=upload_run&redirect_uri=https://YOUR_WEBSITE/auth/splitsio&client_id=YOUR_CLIENT_ID
+    ```
+    If the user authorizes your application, they will be redirected to a URL like
+    ```http
+    https://YOUR_WEBSITE/auth/splitsio?code=YOUR_CODE
+    ```
+    Strip the `code` URL parameter for the next step.
+4. Use your `code` to make this request:
+    ```http
+    POST https://splits.io/oauth/token
+    ```
+    with this body:
+    ```http
+    grant_type=authorization_code
+    client_id=YOUR_CLIENT_ID
+    client_secret=YOUR_CLIENT_SECRET
+    code=YOUR_CODE
+    redirect_uri=http://localhost:8000/auth/splitsio
+    ```
+    which will respond with something like this:
+    ```json
+    {
+      "access_token": "0e82e0ac69fed3c6e5044682a9eb94ccded5ace70e84838104a131cb50595cd2",
+      "token_type": "bearer",
+      "expires_in": 7200,
+      "refresh_token": "0e23b095e0d0104ff0642cde71b61d236f3b3865734a0e734714ecd45b25106c",
+      "scope": "upload_run",
+      "created_at": 1499314941
+    }
+    ```
+    Success! `access_token` is your OAuth token for the user. Use it in your API requests to act on behalf of the user
+    by including this header in your requests:
+    ```http
+    Bearer: OAuth YOUR_ACCESS_TOKEN
```

Comment:
In the api v3 and v4 code it checks for the "Authorization" header, should this be "Authorization" as well or does it check bearer automatically?